### DISCUSSION TITLE
Bug:

### DISCUSSION THREAD

**Discussion by LoafyToast** [FunkinCrew/Funkin#1608] [General]
there is a bug where the camera moves way too far to the left when using a custom character.

**Comment by TentaRJ** [comment]
Please use the template for new issues next time. I don't know what version you were trying to use so I can't help.

**Comment by GStudiosX2** [comment]
Yeah that happens because the offsets by default are crap and u can't add your own ones without editing and it's kinda crap I just suggested a thing to add that
